src/use-format-block.hook.js

```diff
@@ -6,10 +6,44 @@ export function useFormatBlock() {
   return {
     formatBlock(value) {
       if (value === document.queryCommandValue('formatBlock')) {
-        performCommandWithValue('div')
+        if (!!window.chrome && isInHTMLList()) {
```

Comment:
Note that `!!window.chrome` is actually true for all Chromium based browsers, including Opera, Brave, and maybe even safari.

Reply:
I think that's what we want given my observation of safari and chrome.